src/Query/ADQL_parser.hxx

```diff
@@ -1,7 +1,14 @@
 #pragma once
 #include <sstream>
 
+#if 0
+#define BOOST_SPIRIT_DEBUG
+#define BOOST_SPIRIT_DEBUG_OUT std::cout
```

Comment:
Are they referred anywhere?

Reply:
RIght now they're enclosed in an "if 0" block, but if the '0' is changed to a '1', then these defines turn on boost::spirit's debugging mechanism.